example/lib/app/app.dart

```diff
@@ -302,9 +310,78 @@ class _ChewieDemoState extends State<ChewieDemo> {
                 ),
               ],
             ),
+            if (Platform.isAndroid)
```

Comment:
Why not using it for iOS and Android?

Reply:
I don't have any apple device so I can't test on iOS and don't want to modify things that I'm not sure it properly works / is needed.